flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java

```diff
@@ -231,8 +234,9 @@ public void testIllegalStatementInInitFile() throws Exception {
         assertThat(cliClient.executeInitialization(content)).isFalse();
     }
 
-    @Test(timeout = 10000)
-    public void testCancelExecutionInNonInteractiveMode() throws Exception {
+    @Test
+    @Timeout(10000)
```

Comment:
By Default JUnit4 works with millis while JUnit 5 works with seconds...
that's why I would suggest  to specify timeunits since JUnit 5 supports specification of timeunits it would be much more convenient to use them like 
```java
@Timeout(value = 10, unit = TimeUnit.SECONDS)
```
or
```java
@Timeout(value = 10000, unit = TimeUnit.MILLISECONDS)
```